svgReact/icons/index.ts

```diff
@@ -30,7 +30,11 @@ const generateSvgs = ({
     .filter(Boolean)
     /** @desc Preprocessing file */
     .map(item => {
-      const hasSubfolder = item.split('/').length > 1;
+      if (item.split('/').length > 2) {
+        return null;
+      }
```

Comment:
Looks like this logic is to not process more than 2 subfolder level deep. Can you add a comment here?